13.technical-support/04.Reporting-Tool/docs.en.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Report website'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+## Introduction
+
+
+* [Windows](#Windows)
+* [Android](#Android)
+* [MAC](#MAC)
+* [iOS](#iOS)
+* [Filling in the form](#form)
+* [Diagnosis of a issue](#solo)
+
+Form
+
+When using ad blockers, you may face incorrect work of certain websites or broken page elements. These problems might arise due to a variety of factors:
+
+* Incorrect user rules;
+* Modified algorithms of adding ads;
+* Added filters/extensions from third-party developers;
+* The authors of one of the filters made a mistake when creating the rule.
+ 
+Advertisement creators don’t stay put, they’re constantly changing ways to implement ads. A website finds out that an ad was removed and inserts it back, bypassing protection. As a general rule, developers of such sites are testing new algorithms all the time on certain user groups, meanwhile all the rest receive ads where algorithms of the earlier versions were used (to which we’ve provided a solution). This makes it very difficult to find the reason, as this advertising is not displayed by the developers. You can help us by filling out an application using instructions depending on your operating system.
```

Comment:
This makes it very difficult to find the reason, as we don't see new types of ads. You can help us by filling out an application with examples of ads you faced with. To fill the application, please, use instructions for your OS.